File: CovidChartsAndQuiz/app/src/main/java/com/example/myapplication/MainActivity.java
package com.example.myapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void calculateBmi(View view) {
        Intent intent = new Intent(this, BmiActivity.class);
        startActivity(intent);
    }

    public void showCasesChart(View view) {
        Intent intent = new Intent(this, CovidCasesChartActivity.class);
        startActivity(intent);
    }

    public void showRecoveryChart(View view) {
        Intent intent = new Intent(this, CovidRevoveryChartActivity.class);
        startActivity(intent);
    }

    public void showDeathsChart(View view) {
        Intent intent = new Intent(this, CovidDeathsChartActivity.class);
        startActivity(intent);
    }

    public void showQuiz(View view) {
        Intent intent = new Intent(this, QuizActivity.class);
        startActivity(intent);
    }
}

File: CovidChartsAndQuiz-Kotlin/app/src/main/java/com/example/zadanie4/MainActivity.kt
package com.example.zadanie4

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun calculateBmi(view: View) {
        val intent = Intent(this, BmiActivity::class.java)
        startActivity(intent)
    }

    fun showCasesChart(view: View) {
        val intent = Intent(this, CovidCasesChartActivity::class.java)
        startActivity(intent)
    }

    fun showRecoveryChart(view: View) {
        val intent = Intent(this, CovidRevoveryChartActivity::class.java)
        startActivity(intent)
    }

    fun showDeathsChart(view: View) {
        val intent = Intent(this, CovidDeathsChartActivity::class.java)
        startActivity(intent)
    }

    fun showQuiz(view: View) {
        val intent = Intent(this, QuizActivity::class.java)
        startActivity(intent)
    }
}